docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -94,12 +104,23 @@ done
 
 **Configure the kubernetes services on the minion.**
 
-***We need to configure the kubelet and start the kubelet and proxy***
+***We need to configure the kubelet and proxy and start them.***
+
+**Note:** Due to a bug in kubernetes, we must configure an empty JSON authorization file on each minion.
+
+* Create the JSON file by running the following on all minions.
+
+```
+echo "{}" > /var/lib/kubelet/auth
+```
```

Comment:
This should not be needed upstream.  Only in the old version we shipped in RHEL7.